src/main/java/com/commercetools/sync/products/templates/beforeupdatecallback/KeepOtherVariantsSync.java

```diff
@@ -9,7 +9,7 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class KeepOtherVariantsSync {
+public final class KeepOtherVariantsSync {
```

Comment:
Why do you need this breaking change?

Reply:
Added a private constructor for coverage rate.
Let me roll it back and see whether i can implement else classes to raise up the coverage.

Reply:
I just added test case elsewhere to raise up the coverage.
Here I revert the commit.
https://github.com/commercetools/commercetools-sync-java/pull/636/commits/762a065e970d1ae75d0392ffdc0fcbc66bdcc619